Makefile

```diff
@@ -1,33 +1,84 @@
 PROMBENCH_CMD        = /bin/prombench
 DOCKER_TAG = docker.io/sipian/prombench:v2.0.0
 
-deploy:
+create-nodepool:
+	printf ">> Creating NodePools"
 	$(PROMBENCH_CMD) gke nodepool create -a /etc/serviceaccount/service-account.json \
 		-v ZONE:${ZONE} -v PROJECT_ID:${PROJECT_ID} -v CLUSTER_NAME:${CLUSTER_NAME} -v PR_NUMBER:${PR_NUMBER} \
-		-f  components/prombench/nodepools.yaml
+		-f  /prombench/components/prombench/nodepools.yaml
+
+DIR := "/go/src/github.com/${REPO_OWNER}/${REPO_NAME}"
+start-pr:
+	mkdir -p /go/src/github.com
+	printf "\n\n>> Fetching Pull Request"
+	git clone https://github.com/${REPO_OWNER}/${REPO_NAME} ${DIR}
```

Comment:
I don't see any reason why we need these?
lets not use env var unless 100% needed. Just to make it more clear what is happening.

Reply:
These values will not be set as env variables but Makefile variables
ex: `make start-pr REPO_OWNER=prometheus REPO_NAME=prometheus PR_NUMBER=56`

Reply:
yeah my point was why do you need to set them  as args in
```
args:
        - "start-pr"
        - "REPO_OWNER={{ .REPO_OWNER }}"
        - "REPO_NAME={{ .REPO_NAME }}"
        - "PR_NUMBER={{ .PR_NUMBER }}"
```

why not just set them at the beginning of the makefile?

```
REPO_OWNER=prometheus 
REPO_NAME=prometheus
```